examples/blog/src/schema/index.ts

```diff
@@ -8,11 +8,10 @@ import * as Post from './Post'
 import * as Author from './Author'
 
 const appTypes = [Query, Mutation, Blog, Post, Author]
-const nexusPrismaTypes = NexusPrisma.nexusPrismaPlugin({ types: appTypes })
-const allTypes = [appTypes, nexusPrismaTypes]
 
 export default Nexus.makeSchema({
-  types: allTypes,
+  types: appTypes,
+  plugins: [NexusPrisma.create()],
```

Comment:
Or `nexusPrismaPlugin()`.

I expect we're going to have a discussion about if we should enable two ways of doing it, and if only one way, which way.

Reply:
aborted this